ivy_tests/test_ivy/test_frontends/test_jax/test_lax/test_operators.py

```diff
@@ -83,6 +83,15 @@ def test_jax_add(
         x=x[0],
         y=x[1],
     )
+@handle_frontend_test(
```

Comment:
Hey, I don't think your test implementation is quite right. Please refer to the docs or take some inspiration from other tests in this file.

Reply:
Hello @jkeane508 thank you  for the guidance on this function but i made changes and i kindly request for a review,
And also am facing an error while running the test 
**" backend_version = {"version": paddle.version.full_version}
E   AttributeError: module 'paddle' has no attribute 'version'"**